Source/WebCore/html/OffscreenCanvas.h

```diff
@@ -29,6 +29,7 @@
 
 #include "ActiveDOMObject.h"
 #include "AffineTransform.h"
+#include "CSSParserContext.h"
```

Comment:
Would be nice to avoid this include if it’s not needed.

Reply:
Done.